syncplay/messages_fi.py

```diff
@@ -10,120 +10,120 @@
-
-     "media-player-latency-warning": "Varoitus: mediantoisto-ohjelma käytti {} sekuntia vastaamiseen. Mikäli kohtaat yhdentämisongelmia, sulje muita sovelluksia vapauttaaksesi järjestelmävaroja, ja mikäli se ei auta, kokeile jotakin muuta toisto-ohjelmaa.",  # Seconds to respond
-     "mpv-unresponsive-error": "mpv ei ole vastannut käskyihin {} sekuntiin joten vaikuttaa siltä että se ei toimi oikein. Voi olla paras uudelleenkäynnistää Syncplay.",  # Seconds to respond
+     "commandlist-notification/create": "\tc [nimi] - tee tästä huoneesta hallittu",
+     "commandlist-notification/auth": "\ta [salasana] - tunnistaudu operaattoriksi salasanalla",
+     "commandlist-notification/chat": "\tch [viesti] - lähetä viesti huoneeseen",
+     "commandList-notification/queue": "\tqa [tiedosto/osoite] - lisää tiedosto tai osoite soittolistan loppuun",
+     "commandList-notification/queueandselect": "\tqas [tiedosto/osoite] - lisää tiedosto tai osoite soittolistan loppuun, sekä valitse se",
+     "commandList-notification/playlist": "\tql - näytä nykyinen soittolista",
+     "commandList-notification/select": "\tqs [sisällys] - valitse kohde soittolistassa",
+     "commandList-notification/next": "\tqn - valitse toistolistalla seuraava kohde",
+     "commandList-notification/delete": "\tqd [sisällys] - poista valittu kohde soittolistalta",
+     "syncplay-version-notification": "Syncplayn versio: {}",  # syncplay.version
+     "more-info-notification": "Lisätietoja osoitteessa: {}",  # projectURL
+
+     "gui-data-cleared-notification": "Syncplay on tyhjentänyt käyttöliittymän polun ja ikkunan tiedot.",
+     "language-changed-msgbox-label": "Kieli vaihdetaan Syncplayn uudelleenkäynnistyksen yhteydessä.",
+     "promptforupdate-label": "Sallitko Syncplayn tarkistavan päivitykset automaattisesti ajoittain?",
+
+     "media-player-latency-warning": "Varoitus: mediasoittimeltä kesti {} sekuntia vastata. Mikäli kohtaat synkronointi-ongelmia, sulje muita sovelluksia vapauttaaksesi järjestelmäresursseja, ja mikäli se ei toimi, yritä toista mediasoitinta.",  # Seconds to respond
+     "mpv-unresponsive-error": "mpv ei ole vastannut {} sekuntiin, joten se vaikuttaa toimineen virheellisesti. Käynnistä Syncplay uudelleen.",  # Seconds to respond
 
      # Client prompts
-     "enter-to-exit-prompt": "Paina syöttöpainiketta (enter:iä) poistuaksesi\n",
+     "enter-to-exit-prompt": "Paina rivinvaihtonäppäintä (Enter) poistuaksesi\n",
 
      # Client errors
-     "missing-arguments-error": "Joitakin vaadittuja vipuja uupuu, käytä avuksesi --help",
-     "server-timeout-error": "Yhteydenottoaika palvelimeen ylittyi",
-     "mpc-slave-error": "MPC:tä ei saatu käynnistettyä alistustilassa!",
+     "missing-arguments-error": "Joitakin vaadittuja argumentteja uupuu, katso --help",
+     "server-timeout-error": "Aikakatkaisu yhdistäessä palvelimeen",
+     "mpc-slave-error": "MPC:tä ei voitu käynnistetää orjatilassa!",
```

Comment:
```suggestion
     "mpc-slave-error": "MPC:tä ei voitu käynnistetää slave-tilassa!",
```

Thinking on this more, the correct is to keep the original English mode name. I get the impression that mpc would be used by more advanced people who might speak of "masteri" and "slave" anyway